doc/ipam.md

```diff
@@ -0,0 +1,67 @@
+# IPAM REST API
```

Comment:
this file could actually append in core/ipam/doc.go, would be in single place in ipam package ?